src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleline.java

```diff
@@ -105,7 +105,7 @@ void doDoubleBraceInitialization() {
             put("first", "second");
             put("polygene", "lubricants");
             put("alpha", "betical");
-        }}; // it's ok
+        }}; //violation
```

Comment:
Lets have no violations on {{ }} (when INSTANCE_INIT is completely (without any symbols/spaces/.... in OBJ_BLOCK)). We should mention this in xdoc.
Any other cases should be violated in alone mode.

from https://github.com/checkstyle/checkstyle/pull/4247#discussion_r197651247

Reply:
@romani Do you mean an empty `OBJ_BLOCK` when you say _(without any symbols/spaces/.... in OBJ_BLOCK)_ ? Can you please give examples in code.

Reply:
I did a comment over it, here it is(slightly modified):
```
         java.util.Map<String, String> map = new java.util.LinkedHashMap<String, String>() {{
             put("alpha", "betical");
        }}; // it's ok. no way to make this code looks better, it should be special exception from rule.
```

Reply:
@romani 
Yeah I just wasn't so sure about _...completely (without any symbols/spaces/..._. OK will do.

Reply:
Done